src/core/Akka/Configuration/Hocon/HoconTokenizer.cs

```diff
@@ -19,6 +20,17 @@ public class Tokenizer
     {
         private readonly string _text;
         private int _index;
+        private readonly Stack<int> _indexStack = new Stack<int>();
+
+        public void Push()
+        {
+            _indexStack.Push(_index);
+        }
+
+        public void Pop()
+        {
+            _index = _indexStack.Pop();
+        }
```

Comment:
Improved the Hocon parser to let the tokenizer to use transactions.
we can Push the current parser position and then consume some tokens, and then rollback if the consumed text didnt match what we were looking for.

( The Include support is ambigious with normal unquoted hocon keys. and thus we need to "look ahead" to verify if this is actually an include or not)